src/drivers/imu/mpu9250/mpu9250.cpp

```diff
@@ -494,17 +494,17 @@ int MPU9250::reset_mpu()
 		break;
 	case MPU_DEVICE_TYPE_ICM20948:
 		write_reg(ICMREG_20948_PWR_MGMT_1, BIT_H_RESET);
-		usleep(500);  // ICM20948 needs a bit of time here, else register settings will be garbled.
+		/*
+		 * ICM20948 needs a bit of time here, else register settings will be garbled.
+		 */
+	    usleep(200);
```

Comment:
@LorenzMeier 
Without these delays, sensor initialization is getting unstable. I'll check again later to rule out other causes, when things are more stable and/or minimize them. Would you actually stop the task and reschedule during initialization to avoid this?